.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           args: --workspace --all-targets --no-deps -- -D warnings
 
   test:
-    timeout-minutes: 15
+    timeout-minutes: 20
```

Comment:
Is that timeout change still required?

Reply:
I think it can be removed now. Added before rust cache fix.